modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/EditObjectDefinitionMVCRenderCommand.java

```diff
@@ -63,15 +65,17 @@ public String render(
 				_objectDefinitionLocalService.getObjectDefinition(
 					objectDefinitionId));
 			renderRequest.setAttribute(
-				ObjectWebKeys.OBJECT_FIELDS,
-				_objectFieldLocalService.getObjectFields(objectDefinitionId));
-			renderRequest.setAttribute(
-				WebKeys.PORTLET_DISPLAY_CONTEXT,
+				ObjectWebKeys.OBJECT_DEFINITIONS_DETAILS_DISPLAY_CONTEXT,
 				new ObjectDefinitionsDetailsDisplayContext(
```

Comment:
Let's add the renderRequest.setAttribute from the other tabs here, once that is done, I believe there will be no need to have several MVC files. What precautions should we take before removing the other MVC files?